examples/simple-run/index.ts

```diff
@@ -2,7 +2,7 @@ import * as local from "@pulumi/command/local";
 import * as random from "@pulumi/random";
 import { interpolate } from "@pulumi/pulumi";
 
-const pw = new random.RandomPassword("pw", { length: 10 });
+const pw = new random.RandomPassword("pw", { length: 10, special: false });
```

Comment:
Just to be sure, is this to keep unexpected special shell characters from messing up commands in the tests?

Reply:
Yep - that’s right - I saw two test failures caused by this while I was making this change - so just fixed the  test to avoid this.